src/main/java/com/luv2code/ecommerce/service/CheckoutServiceImpl.java

```diff
@@ -40,6 +40,7 @@ public CheckoutServiceImpl(CustomerRepository customerRepository,
     @Transactional
     public PurchaseResponse placeOrder(Purchase purchase) { // method for placing the order
 
+        // comments
         // retrive the order info dto
```

Comment:
E sad, opcenito po kodu na milijun mjesta imas masu komentara. Komentari su NO-NO. Ako su ti komentari potrebni, znaci da ti kod nije citljiv. Pozeljno je imati tocno 0 komentara u kodu, problem su i zbogo ovog sta sam rekao i zbog jos nekig stvari. Koliko vidim koristis relativno deskriptiva imena varijabli, ima nekoliko varijabli s kojima nisam sretan, ali to je sada manje bitno. Probaj takoder kod grupirati u manje cijeline, znaci manje razmaka.